doc/classes/AudioEffectChorus.xml

```diff
@@ -14,78 +14,90 @@
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the cutoff frequency of a specific voice.
 			</description>
 		</method>
 		<method name="get_voice_delay_ms" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the delay of a specific voice in milliseconds.
 			</description>
 		</method>
 		<method name="get_voice_depth_ms" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the depth of a specific voice filter in milliseconds.
 			</description>
 		</method>
 		<method name="get_voice_level_db" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the volume of a specific voice in decibels.
 			</description>
 		</method>
 		<method name="get_voice_pan" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the pan level of a specific voice.
```

Comment:
This is a bit vague. I am to assume that the returned value is a number between `-1.0` and `1.0`, but it is not explained.

Reply:
Thanks for the feedback. We put ranges for the setters but not for the getters, my thinking being that it made more sense to tell folks the recommended range in which they'd be setting. If putting them with the getters as well is best practice we can do that.

Reply:
Indeed. When a value is returned, the user must know what the value means.
```suggestion
				Gets the pan level of a specific voice, as a value between `-1.0` (left ear) and `1.0` (right ear).
```
Alternatively, if the ear is deemed obvious.
```suggestion
				Gets the pan level of a specific voice, as a value between `-1.0` and `1.0`.
```